src/script/page/MainContent/panels/preferences/avPreferences/DeviceSelect.tsx

```diff
@@ -59,15 +59,16 @@ const DeviceSelect: React.FC<DeviceSelectProps> = ({
       <div className="preferences-option-icon preferences-av-select-icon">
         <DeviceIcon />
       </div>
+
       <div css={{width: '550px'}}>
         <Select
           id={uieName}
-          onChange={onChange}
+          onChange={option => onChange(option?.value as string)}
```

Comment:
same and id rename the function but you can merge it
```suggestion
          onChange={option => onChange(option?.value.toString())}
```